npg-api/npg-notification-api.yaml

```diff
@@ -0,0 +1,81 @@
+
+openapi: 3.0.3
+info:
+  title: pagoPA Wallet Notification API
+  version: "0.1"
+  description: |-
+    Notification API to implement callback providing order status updates after wallet creation on Nexi's NPG
+externalDocs:
+  description: Nexi API to handle Hosted Payment Page (HPP) payments
+  url: https://developer.nexigroup.com/it/api/post-orders-hpp
+paths:
+  /notify:
+    post:
+      summary: Update Wallet on NPG payment instrument creation
+      description: Update Wallet on NPG payment instrument creation
+      operationId: notifyWallet
+      parameters:
+        - in: header
+          name: Correlation-Id
+          schema:
+            type: string
+            format: uuid
+          required: true
+          description: Unique identifier of the the request
```

Comment:
type: double "the"

Reply:
Fixed in [db82361](https://github.com/pagopa/pagopa-wallet-service/pull/18/commits/db82361accc26a17b5b66337457827422c0aef1f)